build.gradle

```diff
@@ -70,6 +70,8 @@ dependencies {
 
 	compile("uk.gov.ons.census.fwmt:census-fwmt-events:1.0.17-RELEASE")
 	compile("uk.gov.ons:census-fsdr-common:0.0.4-SNAPSHOT")
+	compile("uk.gov.ons.census.ffa:census-ffa-storage-utils:0.0.1-SNAPSHOT")
```

Comment:
these need to be the release versions

Reply:
Do not think there are release versions

Reply:
was gonna change when there was (currently have them locally)